test/test_corpus.cc

```diff
@@ -836,7 +836,7 @@ void testDocumentSymbols(LSPWrapper &lspWrapper, Expectations &test, int &nextId
 }
 
 TEST_P(LSPTest, All) {
-    string rootPath = "/Users/jvilk/stripe/pay-server";
+    string rootPath = fmt::format("/Users/{}/stripe/sorbet", std::getenv("USER"));
```

Comment:
Any particular reason you changed this? There shouldn't be any reason for this to be a real path on the file system.

Reply:
You're totally right that there's no *need* for it, but uris constructed from this showed up in test failure logs, and VS Code was dutifully highlighting them, so at least for myself, it made the highlighting a little more useful.